ibis/backends/e6data/__init__.py

```diff
@@ -0,0 +1,573 @@
+"""The E6Data backend."""
+
+from __future__ import annotations
+
+import contextlib
+import re
+import warnings
+from functools import cached_property, partial
+from itertools import repeat
+from operator import itemgetter
+from typing import TYPE_CHECKING, Any
+from urllib.parse import parse_qs, urlparse
+
+import numpy as np
+import pymysql
```

Comment:
Are you using this somewhere? If not, pleas remove it as an e6data backend dependency and also remove this import.